app/controllers/admin/playlists_controller.rb

```diff
@@ -7,5 +7,13 @@ def authorize_resource
       @resource = Student.find(params[:student_id]).playlist
       authorize(resource)
     end
+
+    def collection
+      Playlist.includes(:student)
+              .where(users: {grove_id: current_user.grove.id})
+              .where('name ILIKE ?', "%#{params[:student_search]}%")
```

Comment:
the `%` symbol is the wildcard. Also, this search does work when the `student_search` field is blank; it returns all Playlists.